src/Containers/Piano.jsx

```diff
@@ -60,32 +82,21 @@ export default class Piano extends React.Component {
   }
 
   selectNaturals() {
-    const stateChange = {
-      "C": true,
-      "D": true,
-      "E": true,
-      "F": true,
-      "G": true,
-      "A": true,
-      "B": true,
-    };
+    const stateChange = _.chain(naturals)
+      .keyBy((key) => key)
```

Comment:
It is nice that it is explicit, but FYI keyBy defaults to using _.identity, so not providing a function would do the same